src/test/java/games/strategy/internal/persistence/serializable/VersionProxyAsProxyTest.java

```diff
@@ -0,0 +1,24 @@
+package games.strategy.internal.persistence.serializable;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import games.strategy.persistence.serializable.AbstractProxyTestCase;
+import games.strategy.persistence.serializable.ProxyFactory;
+import games.strategy.util.Version;
+
+public final class VersionProxyAsProxyTest extends AbstractProxyTestCase<Version> {
+  public VersionProxyAsProxyTest() {
+    super(Version.class);
+  }
+
+  @Override
+  protected Version createPrincipal() {
+    return new Version(1, 2, 3, 4);
```

Comment:
Might make sense for this method to return a `Collection` of `Version` objects. In other cases it may be very interesting to test different constructors, and perhaps for `Version` as well since it too has multiple constructors. I suspect in testing we would typically want to cover each constructor available.